services/pub/pub-publisher.service.js

```diff
@@ -0,0 +1,49 @@
+import Joi from 'joi'
+import { BaseJsonService } from '../index.js'
+
+const schema = Joi.object({
+  publisherId: Joi.string().allow(null).required(),
+}).required()
+
+class PubPublisher extends BaseJsonService {
+  static category = 'other'
+
+  static route = {
+    base: 'pub/publisher',
+    pattern: ':packageName',
+  }
+
+  static examples = [
+    {
+      title: 'Pub Publisher',
+      namedParams: { packageName: 'example' },
```

Comment:
Normally we use a real package for the examples on the home page. Can we replace this with a real package that has a verified publisher

Reply:
Gotcha - I'll update this to `dart.dev`.